spec/lib/concerns/coa_concourse/build_spec.rb

```diff
@@ -40,6 +40,54 @@
     end
   end
 
-  describe '#handle_result'
+  describe '#handle_result' do
+    let(:build) { described_class.new(name: 1, status: status, full_job_name: "p1/j1") }
+
+    context "when a failure happened" do
+      let(:status) { "errored" }
+
+      context "when the failure is ignored"  do
+        it "just logs the final status" do
+          logger = build.logger
+          allow(logger).to receive(:log_and_puts)
+
+          build.handle_result(true)
+
+          expect(logger).to have_received(:log_and_puts).once.
+            with(:info, "Final status for job 'p1/j1': errored")
+          expect(logger).to have_received(:log_and_puts).once.
+            with(:info, "Failure ignored for this job.")
+        end
+      end
+
+      context "when the failure is not ignored" do
```

Comment:
The code related to this test seems to generate the issue, according to : 
 - https://github.com/rspec/rspec-core/issues/2235
 - https://github.com/rspec/rspec-core/issues/2342